cc/tflite_micro/tools/generate_cc_arrays.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
If this was copied (even in part) from the TF repo, it must go under third_party (and not have our copyright)

Reply:
Let me think how to do it as generated clean TFLM sources doesn't include the tool script and we're and will be updating the script to support more formats afterwards afterwards.
That means every time we upgrade/downgrade TFLM, our script needs to copy the tool and its BUILD file to somewhere, after finishing upgrade/downgrade process restoring them back to TFLM source tree.

Reply:
Done by updating our script to backup/restore it in third_party/tflite-micro